disputer/disputer.js

```diff
@@ -2,7 +2,15 @@
 // wallet to run the liquidations. Future versions will deal with generating additional synthetic tokens from EMPs as the bot needs.
 
 class Disputer {
-  constructor(logger, expiringMultiPartyClient, gasEstimator, account) {
+  /**
+   * @notice Constructs new Disputer bot.
+   * @param {Object} logger Module used to send logs.
+   * @param {Object} expiringMultiPartyClient Module used to query EMP information on-chain.
+   * @param {Object} gasEstimator Module used to estimate optimal gas price with which to send txns.
+   * @param {String} account Ethereum account from which to send txns.
+   * @param {Object} [config] Contains fields with which constructor will attempt to override defaults.
```

Comment:
what does the `[ ]` represent around the config here? that it's an object/ array or optional?

Reply:
means its an optional param